tests/models/test_resnet.py

```diff
@@ -42,12 +42,36 @@ def mocked_weights(
             monkeypatch.setattr(weights, 'url', str(path))
         return weights
 
+    @pytest.fixture
+    def mocked_weights_features_only(
```

Comment:
Can we combine these two tests so there is less duplicated code? Something like:
```python
@pytest.fixture(params=[True, False])
def mocked_weights(..., request: SubRequest):
    features_only = request.param
```